docs/Parameters.md

```diff
@@ -54,8 +54,8 @@ The parameter format is `key1=value1 key2=value2 ... ` . And parameters can be s
   * support multi validation data, separate by `,`
 * `num_iterations`, default=`100`, type=int, alias=`num_iteration`,`num_tree`,`num_trees`,`num_round`,`num_rounds`
   * number of boosting iterations
-  * note: `num_tree` here equal with `num_iterations`. For multi-class, it actually learns `num_class * num_iterations` trees.
-  * note: For python/R package, cannot use this parameters to control number of iterations.
+  * note: For python/R package, **this parameters is ignored**.
+  * note: internally, LightGBM constructs `num_class * num_iterations` trees for `multiclass` problems. Neither of aliases of this parameter directly control number of actually constructed trees.
```

Comment:
Put back the previous note for this line, as the current note is more ambiguous than the previous one (the previous one is technically correct, the current one comes out of the blue).